examples/resources/cdp_dw_aws_cluster/resource.tf

```diff
@@ -0,0 +1,54 @@
+## Copyright 2024 Cloudera. All Rights Reserved.
+#
+# This file is licensed under the Apache License Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. Refer to the License for the specific
+# permissions and limitations governing your use of the file.
+
+resource "cdp_dw_aws_cluster" "example" {
+  crn  = cdp_datalake_aws_datalake.example.crn
+  name = "<value>"
+  cluster_id = "<value>"
+  cloud_platform = "AWS",
+  node_role_cdw_managed_policy_arn = "<value>",
+  reduced_permission_mode = false,
```

Comment:
I'm not sure if we'd need this. We'll be unable to provision reduced perm cluster via TF

Reply:
set to static false in the create call